### ISSUE TITLE
Hot module replacement support

### ISSUE THREAD

**Issue by nukeop** [gabiseabra/google-fonts-webpack-plugin#2]
Hi, great work on the plugin, it makes my life a lot easier.

I was wondering if it'd be hard to add support for hot module replacement, as right now with each hot reload the plugin builds the fonts once again and adds them to the bundle over and over, which forces me to restart the webpack server every now and again (because it gets slower and slower). Would you be able to solve that somehow?

**Comment by gabiseabra** [comment]
Thanks, I'll be marking this as a feature request 👍
The solution for now is using `local: false` in development, but that only works with html-webpack-plugin.

**Comment by debanjanbasu** [comment]
Yeah, this is a major issue for my development. Can we add this feature ASAP. This is an awesome plugin otherwise.
